src/Faker/Provider/et_ET/Person.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Faker\Provider\et_ET;
+
+class Person extends \Faker\Provider\Person
+{
+
+
+    protected static $titleMale = array('Härra', 'Hr.');
+
+    protected static $titleFemale = array('Proua', 'Pr.', 'neiu', 'preili');
+
+    /**
+     * @link http://www.eki.ee/books/ekk09/index.php?p=6&p1=5&id=548
```

Comment:
This is an array of names which cannot be under any copyright by default. The source materials on the link (research into names, etc) does have the authors' copyright marked.
If you are worried that a list of names is under copyright, then I propose the link be removed instead.